ibc_public/utils_labels.py

```diff
@@ -0,0 +1,40 @@
+import json
+import sys
+import pandas as pd
+
+import os
+
+_package_directory = os.path.dirname(os.path.abspath(__file__))
+
+ALL_CONTRASTS = os.path.join(_package_directory, '..', 'ibc_data',
+                             'all_contrasts.tsv')
+CONTRAST_COL = 'contrast'
+
+
+def get_labels(*contrasts: str, contrast_col=CONTRAST_COL):
+    """Returns the list of labels for each passed contrast name"""
```

Comment:
Please provide a complete docstring.

Reply:
+ I think that  the function should return a dictionary with contrasts as indexes and labels as values.

Reply:
Done